src/main/java/io/vlingo/lattice/model/object/ObjectEntity.java

```diff
@@ -280,7 +280,9 @@ final public void queryObjectResultedIn(
           final Object object) {
     outcome
       .andThen(result -> {
-        stateObject((T) queryResult.stateObject);
+        final T stateObject = (T) queryResult.stateObject;
+        this.id = String.valueOf(stateObject.persistenceId());
```

Comment:
@VaughnVernon is this something that needs to be done also for other entity types or is `ObjectEntity` the only one that supports db generated ids?